#goal definition
nav_msgs/Path waypoints
float64[] times
geometry_msgs/TwistStamped initVel
geometry_msgs/TwistStamped finalVel
geometry_msgs/AccelStamped initAcc
geometry_msgs/AccelStamped finalAcc
---
#result definition
bool ok
---
#feedback definition
float64 completePerc
